src/angular-localForage.js

```diff
@@ -143,7 +143,11 @@
           var copy;
           if (angular.isArray(value)) {
             return value.map(stripMeta);
-          } else if (angular.isObject(value) && !(value instanceof Blob)) {
+          } 
+          else if(angular.isObject(value) && (value instanceof Date)) {
+            return value;
+          }
+          else if (angular.isObject(value) && !(value instanceof Blob)) {
```

Comment:
Style nit: Will you please put the `else`s and `else if`s on the same line as the closing brace of the `if` and make sure there is a space between `else`/`else if` and the opening `(` of the conditional statement? Like:

```js
} else if (false === true) {
```